apps/stack/template.yaml

```diff
@@ -310,7 +359,19 @@ Resources:
         - !Ref Topic
       Parameters:
         BucketArn: !GetAtt Bucket.Arn
-        FilterUri: !Ref MetricStreamFilterUri
+        FilterUri: !If
+          - DeployLambda
+          - 's3://observeinc/cloudwatchmetrics/filters/default.yaml'
+          - !Ref MetricStreamFilterUri
+          # default.yaml must exist. When using a lambda for config we use
```

Comment:
# We provide a yaml file URI here where the contents are a sentinel value that the user would never enter.
# Then the lambda modifies the metricstream to reflect the actual config provided.
# We need this sentinel value here initially so that if we switch back to the yaml based approach,
# the cloudformation stack will recognize that the value from the yaml file has changed and will update.